app/default_overrides.go

```diff
@@ -162,7 +162,7 @@ func (govModule) DefaultGenesis(cdc codec.JSONCodec) json.RawMessage {
 	oneWeek := day * 7
 	twoWeeks := oneWeek * 2
 
-	genState.DepositParams.MinDeposit = sdk.NewCoins(sdk.NewCoin(BondDenom, sdk.NewInt(1_000_000_000))) // 1000 TIA
+	genState.DepositParams.MinDeposit = sdk.NewCoins(sdk.NewCoin(BondDenom, sdk.NewInt(10_000_000_000))) // 10000 TIA
```

Comment:
[blocking] `10_000_000_000` looks incorrect because `10_000_000_000 / 10e6 = 1_000`. Prior to this change, MinDeposit was `1_000_000_000` which looks like 100 TIA not 1000 TIA.

Ref: 1_000_000_000 / 10e6

@evan-forbes can you fact check me here? Prior to this change, I think it should have been `10_000_000_000` and after this change I think it should be `100_000_000_000`.



[nit] comma for readability

```suggestion
	genState.DepositParams.MinDeposit = sdk.NewCoins(sdk.NewCoin(BondDenom, sdk.NewInt(100_000_000_000))) // 10,000 TIA
```

Reply:
Although https://testnet.mintscan.io/celestia-testnet/parameters has the correct 1,000 TIA 

## Screenshot

<img width="288" alt="Screenshot 2023-10-09 at 9 46 12 AM" src="https://github.com/celestiaorg/celestia-app/assets/3699047/1d89016d-67d1-48e5-bb8a-d29e2e273659">

Reply:
we're good, 1_000_000_000 is 1000, just remove 6 0s

Reply:
oof you're right, I miscalculated, resolving.